storage/objects/change_file_storage_class.go

```diff
@@ -38,14 +38,23 @@ func changeObjectStorageClass(w io.Writer, bucket, object string) error {
 	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
 	defer cancel()
 
-	bkt := client.Bucket(bucket)
-	obj := bkt.Object(object)
+	o := client.Bucket(bucket).Object(object)
+
+	// Set a generation-match precondition. The request to upload is aborted
```

Comment:
I'm wondering if we should note that this is optional. What do others think? @cbonnie @shaffeeullah

Reply:
Sounds good to me.